src/components/Article.jsx

```diff
@@ -1 +1,17 @@
-// export const Article = () => ();
+import React from 'react';
+
+const Article = ({ title, text }) => (
```

Comment:
you have missing prop
```suggestion
const Article = ({ title, text, date }) => (
```